sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1673,6 +1677,8 @@ public partial class BlockBlobClient : Azure.Storage.Blobs.Specialized.BlobBaseC
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlockList>> GetBlockListAsync(Azure.Storage.Blobs.Models.BlockListTypes blockListTypes = Azure.Storage.Blobs.Models.BlockListTypes.All, string snapshot = null, Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.IO.Stream OpenWrite(bool overwrite, Azure.Storage.Blobs.Models.BlockBlobOpenWriteOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<System.IO.Stream> OpenWriteAsync(bool overwrite, Azure.Storage.Blobs.Models.BlockBlobOpenWriteOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        protected static Azure.Response<Azure.Storage.Blobs.Models.BlobContentInfo> PutBlob(Azure.Storage.Blobs.Specialized.BlockBlobClient client, System.IO.Stream content, Azure.Storage.Blobs.Models.BlobUploadOptions options, string operationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        protected static System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobContentInfo>> PutBlobAsync(Azure.Storage.Blobs.Specialized.BlockBlobClient client, System.IO.Stream content, Azure.Storage.Blobs.Models.BlobUploadOptions options, string operationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I'm going to remove these. But we actually hid the PutBlob API, because we we prefer customers to use the UploadAsync call and have the SDK determine if we should chunk the blob or just do one big put blob call.